### PR TITLE
Feature/twitter url

### PR DESCRIPTION
**Pull request by crbrox**
Allow to change twitter endpoint by configuration. (Useful for testing mainly)
Not documented intentionally, it should not be used in normal operation

@fgalan 
@dmoranj 
@jcalderin

### CODE REVIEW

File: npm-shrinkwrap.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "perseo",
-  "version": "0.4.1-next",
+  "version": "0.5.0-next",
```
Comment (fgalan) [inline npm-shrinkwrap.json]: Having adjusted this "by hand" probably mean that the release.sh stepping script should take also npm-shrinkwrap.json into account and currently it is not doing it.

@dmoranj could you have a look this? Maybe it also affect to fiware-orion-pep...

Comment (dmoranj) [inline npm-shrinkwrap.json]: We are not currently using the shrinkwrap in the PEP proxy (maybe we should), so this problem won't reproduce there. I'll modify the release script to fix this.

Comment (dmoranj) [inline npm-shrinkwrap.json]: Created issue #100 to separate this problem from the actual PR.

**Comment by dmoranj** [comment]
LGTM

**Pull request merged**